addons/web/static/src/views/calendar/calendar_common/calendar_common_renderer.js

```diff
@@ -221,7 +221,14 @@ export class CalendarCommonRenderer extends Component {
     onEventClick(info) {
         this.click(info);
     }
-    onEventContent({ event }) {
+    onEventContent(arg) {
+        const { event } = arg;
+        if (event.start && event.end) {
+            const timeFormat = is24HourFormat() ? "H:mm" : "h:mm a";
```

Comment:
This is the 3rd occurrence in the same file of the same line (but yours is a bit different).
I'd rather factorize on a new class member: i.e. in setup `this.timeFormat = is24HourFormat() ? "HH:mm" : "hh:mm a";`

Or we could use Intl.NativeDateFormat.

Is is [green baseline](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat), [supported by luxon](https://moment.github.io/luxon/#/formatting?id=intldatetimeformat) and [supported/augmented by fullcalendar](https://fullcalendar.io/docs/date-formatting).